### ISSUE TITLE
Cannot parse key file created by flipper himself

### ISSUE THREAD

**Issue by Salamafet** [flipperdevices/flipperzero-firmware#2279]
### Describe the bug.

According to this [thread](https://forum.flipperzero.one/t/cannot-parse-file-created-from-the-flipper-sub-ghz-read-menu/11317), I can parse a sub-ghz key file create with the Flipper himself.

Here two example button of the same remote:
```
Filetype: Flipper SubGhz Key File
Version: 1
Frequency: 433920000
Preset: FuriHalSubGhzPresetOok650Async
Protocol: CAME
Bit: 42
Key: 00 00 03 77 BE E9 8D 0B
```
```
Filetype: Flipper SubGhz Key File
Version: 1
Frequency: 433920000
Preset: FuriHalSubGhzPresetOok650Async
Protocol: CAME
Bit: 42
Key: 00 00 03 77 BF 68 2C 13
```

### Reproduction

1. Read the key from the Sub-ghz menu
2. Save the key
3. Replay the saved key

Get the "Cannot parse file" error
![](https://forum.flipperzero.one/uploads/default/original/2X/d/d17ac9f8111e76fd8090039f387656e8919f5538.png)

**Comment by Skorpionm** [comment]
when did you create this file?  exactly this file was created on the firmware?  for the CAME protocol, I know 12 bits 23 24 25. It cannot be 42 bits or it is some other protocol.

**Comment by Salamafet** [comment]
Create with the firmware 0.74.2 and got the error with 0.74.2 and 0.75.

Here is 4 photos of the remote
[photos_remote.pdf](https://github.com/flipperdevices/flipperzero-firmware/files/10400668/remote.pdf)

Screenshots of the Flipper screen during the normal Read
![sg](https://user-images.githubusercontent.com/783351/212045193-35554f41-4923-4316-bb6c-33e2600b5d14.png)
![sg_1](https://user-images.githubusercontent.com/783351/212045186-d646ead2-c33a-4757-8cb0-1192872cebef.png)
![sg_2](https://user-images.githubusercontent.com/783351/212045190-32f83f7d-2dd6-4f67-bbb2-8c21faa18db0.png)

Raw files:

Button 1:
```
Filetype: Flipper SubGhz RAW File
Version: 1
Frequency: 433657070
Preset: FuriHalSubGhzPresetOok650Async
Protocol: RAW
RAW_Data: 9849 -570 165 -300 233 -132 131 -1000 131 -4304 65 -64 231 -300 165 -66 393 -68 463 -100 863 -100 1065 -66 331 -100 665 -134 369 -66 337 -100 14517 -136 265 -166 163 -202 959 -66 627 -66 335 -68 669 -166 437 -66 3393 -68 331 -68 297 -66 8311 -236 237 -134 65 -134 231 -132 65 -164 99 -66 231 -66 131 -826 67 -132 65 -3618 67 -132 97 -162 97 -390 459 -100 333 -100 199 -136 167 -100 197 -68 1593 -68 695 -66 403 -168 661 -100 197 -136 333 -960 65 -68 163 -1218 97 -1160 233 -166 133 -98 229 -98 525 -132 231 -100 199 -932 229 -166 65 -136 469 -102 133 -66 233 -68 769 -66 1583 -164 599 -100 299 -68 7593 -100 535 -698 297 -98 199 -134 65 -66 65 -200 97 -264 99 -266 165 -134 201 -164 133 -430 65 -690 97 -168 201 -266 67 -66 661 -1490 1483 -66 1411 -238 133 -132 199 -864 165 -68 65 -66 165 -200 65 -136 229 -3060 65 -98 65 -1988 99 -132 131 -268 235 -430 231 -134 65 -134 167 -168 267 -100 297 -166 399 -132 97 -100 401 -200 165 -332 265 -98 235 -134 497 -166 1933 -68 7287 -398 129 -164 133 -198 195 -228 197 -396 131 -6618 165 -100 233 -134 265 -134 267 -134 827 -98 435 -98 167 -66 269 -100 1193 -100 425 -100 327 -98 231 -68 2111 -130 7307 -430 65 -424 65 -132 65 -1024 99 -3888 197 -194 99 -66 231 -166 333 -66 999 -132 99 -68 99 -100 197 -98 327 -98 163 -66 65 -66 199 -100 331 -68 633 -166 395 -100 7311 -266 163 -368 165 -202 433 -66 165 -200 499 -262 97 -4652 165 -198 131 -264 163 -98 883 -98 2115 -100 631 -100 1391 -100 1251 -66 6713 -366 263 -1060 231 -134 365 -166 133 -6238 197 -464 65 -98 261 -162 65 -98 163 -98 195 -494 625 -200 65 -100 497 -164 529 -132 65 -130 817 -66 259 -64 691 -136 697 -98 7771 -398 263 -66 399 -338 99 -596 133 -166 267 -332 327 -66 661 -100 265 -136 265 -134 301 -1132 231 -98 235 -400 197 -200 165 -200 231 -202 99 -232 329 -100 231 -168 1129 -7302 65 -132 65 -894 97 -566 65 -66 199 -562 263 -198 129 -266 297 -266 133 -132 67 -132 67 -166 165 -66 363 -66 97 -330 333 -100 395 -98 1027 -102 265 -66 1851 -98 947 -66 5577 -4030 165 -166 263 -132 325 -132 131 -164 263 -760 493 -164 519 -98 1353 -4094 267 -300 197 -430 65 -134 133 -132 265 -102 301 -166 65 -134 1229 -132 131 -132 267 -232 765 -68 463 -66 299 -132 333 -98 263 -68 363 -100 267 -100 727 -68 11923 -66 97 -396 99 -100
RAW_Data: 233 -202 433 -528 427 -98 261 -296 797 -198 4143 -66 12747 -392 97 -424 163 -594 295 -200 99 -432 199 -562 65 -166 101 -334 165 -200 497 -368 559 -300 197 -300 395 -68 265 -66 3587 -68 997 -66 631 -298 129 -98 231 -98 65 -196 393 -264 131 -66 165 -296 131 -198 63 -558 65 -2878 197 -230 63 -362 99 -166 297 -132 197 -100 1065 -100 165 -134 901 -66 15347 -66 63 -394 465 -66 199 -428 99 -98 393 -396 17411 -102 197 -560 295 -858 195 -230 97 -962 527 -100 865 -232 659 -328 199 -166 199 -366 65 -98 4055 -134 299 -130 197 -68 301 -134 235 -102 1023 -66 8433 -192 131 -6372 197 -98 129 -130 65 -98 197 -66 131 -98 99 -296 295 -64 195 -1182 329 -526 163 -594 65 -1054 329 -66 735 -98 7275 -166 165 -960 197 -236 99 -430 231 -132 165 -5488 65 -1252 131 -230 163 -232 227 -162 331 -198 65 -100 333 -168 195 -1196 233 -202 397 -66 133 -166 1397 -100 6865 -2810 65 -2028 131 -858 165 -300 135 -200 167 -132 133 -198 233 -166 695 -134 235 -166 925 -100 5051 -100 491 -64 195 -66 425 -230 915 -564 99 -66 167 -1064 165 -134 233 -1148 265 -434 297 -264 361 -232 333 -134 397 -132 263 -100 195 -98 163 -100 163 -132 229 -164 197 -98 229 -132 229 -202 1267 -100 633 -100 6097 -198 131 -164 329 -66 327 -1706 495 -134 231 -134 465 -198 165 -134 299 -134 333 -930 165 -398 131 -164 231 -132 995 -100 267 -132 367 -134 1363 -68 8719 -66 131 -396 99 -662 197 -526 99 -1478 263 -132 231 -820 165 -492 229 -332 229 -1256 165 -166 133 -100 67 -134 131 -166 433 -200 401 -132 231 -166 12383 -764 99 -566 199 -198 99 -132 233 -66 533 -300 99 -400 893 -330 165 -66 131 -166 201 -166 295 -96 591 -364 697 -100 99 -100 559 -66 261 -264 295 -264 295 -100 495 -134 201 -266 12297 -794 65 -166 133 -1518 167 -702 65 -100 229 -202 429 -232 265 -66 265 -132 331 -98 199 -102 301 -400 99 -598 197 -98 101 -366 301 -332 493 -100 361 -132 265 -164 97 -66 8139 -4070 259 -730 365 -134 331 -66 267 -134 265 -234 297 -98 199 -498 397 -460 391 -100 97 -66 65 -460 231 -234 397 -132 425 -66 749 -164 457 -66 10969 -786 65 -98 131 -526 97 -98 163 -296 65 -688 265 -98 395 -100 197 -170 565 -166 229 -134 99 -100 97 -66 233 -198 331 -1064 67 -198 901 -308 657 -314 649 -294 669 -316 661 -300 649 -302 661 -320 661 -300 647 -300 659 -320 663 -300 649 -310 655 -314 659 -294 665 -318 629 -5150 641 -290
RAW_Data: 685 -314 625 -340 651 -296 663 -316 629 -304 685 -296 663 -322 631 -292 669 -312 655 -312 651 -294 665 -312 665 -320 645 -290 643 -5120 677 -320 643 -288 681 -314 625 -340 651 -312 623 -316 663 -294 667 -312 661 -312 649 -294 667 -310 665 -282 671 -290 669 -304 659 -320 631 -4818 315 -638 317 -622 345 -280 685 -628 321 -640 303 -630 351 -278 663 -644 315 -636 351 -600 329 -650 311 -306 665 -622 323 -644 309 -634 351 -626 323 -642 317 -320 625 -640 353 -622 331 -622 345 -268 669 -622 323 -316 649 -310 657 -312 649 -294 665 -624 323 -316 643 -310 657 -312 659 -312 623 -316 649 -644 311 -628 343 -626 349 -626 293 -636 341 -312 641 -316 661 -298 647 -314 655 -24180 311 -626 341 -614 347 -314 639 -622 329 -650 313 -622 341 -322 633 -622 355 -630 323 -642 313 -622 341 -322 643 -608 351 -628 323 -638 319 -628 325 -642 315 -312 661 -622 321 -628 349 -624 335 -320 627 -646 311 -322 631 -326 645 -302 661 -320 633 -620 349 -280 661 -320 633 -328 645 -304 661 -320 641 -610 351 -602 321 -636 347 -624 339 -622 345 -302 641 -318 627 -328 647 -312 623 -24208 355 -594 325 -644 309 -304 667 -620 323 -646 313 -658 297 -352 623 -646 309 -658 293 -636 349 -602 329 -350 603 -648 353 -598 323 -632 347 -624 337 -620 343 -302 645 -654 287 -650 315 -658 297 -352 621 -648 309 -320 631 -328 641 -302 663 -318 633 -620 349 -312 633 -318 633 -328 645 -300 663 -318 635 -618 343 -636 309 -640 321 -660 315 -634 317 -312 633 -350 637 -290 665 -304 631 -24202 317 -636 317 -628 329 -314 661 -614 339 -624 343 -626 347 -288 663 -608 351 -602 321 -634 349 -624 339 -322 633 -618 353 -630 319 -642 313 -622 339 -624 345 -302 639 -622 321 -648 311 -666 319 -320 633 -620 355 -318 625 -302 649 -310 657 -314 649 -612 343 -322 641 -290 647 -352 621 -302 649 -312 655 -648 313 -622 339 -624 343 -624 345 -594 327 -310 669 -312 655 -310 623 -346 621 -24202 323 -630 317 -630 323 -310 657 -614 339 -622 343 -624 347 -320 633 -616 351 -624 289 -654 349 -622 333 -320 609 -646 347 -624 293 -670 317 -628 325 -644 309 -304 669 -620 321 -646 313 -666 317 -320 633 -616 343 -314 639 -318 627 -328 643 -310 655 -642 315 -310 659 -312 623 -340 615 -344 621 -312 647 -648 309 -640 319 -628 339 -632 345 -604 321 -334 649 -294 669 -320 629 -292 667 -24186 319 -622 349 -622 335 -320 627 -614 341 -648 317 -638 319 -318 625 -640 349 -624 333 -622
RAW_Data: 345 -620 311 -322 661 -612 349 -602 321 -632 345 -634 319 -632 325 -312 639 -646 351 -600 329 -644 315 -310 663 -624 325 -310 637 -322 663 -304 627 -352 635 -614 351 -280 661 -320 633 -326 641 -304 661 -318 635 -618 353 -628 319 -620 351 -622 333 -620 345 -270 669 -318 629 -328 647 -312 623 -24204 355 -594 325 -646 311 -304 667 -640 289 -650 313 -666 319 -318 631 -652 319 -628 323 -644 311 -656 309 -320 625 -644 347 -612 325 -642 315 -660 297 -648 309 -304 667 -620 321 -646 313 -634 351 -318 631 -620 353 -318 625 -302 651 -310 655 -310 647 -648 309 -320 625 -324 645 -316 659 -298 647 -310 655 -646 311 -622 339 -612 317 -670 319 -628 325 -310 657 -294 667 -318 629 -324 641 -24186 319 -624 349 -624 333 -320 611 -644 349 -626 295 -634 353 -320 623 -636 351 -624 297 -650 345 -620 311 -322 639 -646 319 -628 321 -644 317 -626 331 -648 311 -306 667 -624 321 -644 311 -632 351 -322 631 -616 355 -320 623 -302 649 -310 655 -314 659 -614 337 -322 627 -292 671 -318 629 -328 641 -312 655 -612 347 -624 337 -622 345 -622 311 -628 343 -304 645 -318 661 -296 649 -304 629 -136878 329 -328 97 -132 129 -360 129 -230 97 -326 333 -100 697 -98 295 -198 265 -298 197 -464 197 -100 233 -362 131 -98 197 -200 727 -98 979 -134 6471 -1176 129 -5024 99 -1892 99 -368 263 -166 2623 -132 929 -100 693 -100 63 -166 9409 -1098 99 -1760 65 -3520 65 -66 99 -1652 197 -132 97 -164 65 -426 229 -100 97 -264 99 -260 361 -66 489 -66 457 -100 553 -132 227 -164 325 -100 427 -98 327 -66 295 -130 715 -164 813 -66 425 -98 693 -64 11125 -390 63 -98 229 -328 197 -166 231 -664 297 -64 263 -64 231 -328 553 -98 63 -130 97 -98 493 -100 8239 -438 97 -700 265 -6704 133 -134 67 -234 99 -98 99 -132 133 -132 99 -200 97 -200 895 -300 233 -300 199 -134 303 -200 97 -166 65 -200 299 -100 263 -202 133 -100 131 -66 497 -100 763 -66 267 -100 363 -68 11573 -1622 263 -164 67 -1754 299 -98 229 -64 655 -130 63 -196 131 -66 603 -66 267 -134 333 -132 233 -132 97 -66 497 -134 263 -66 7851 -826 133 -164 99 -462 131 -166 101 -368 131 -434 97 -134 365 -1358 67 -1524 197 -526 329 -428 129 -328 265 -198 163 -196 1079 -64 1085 -198 691 -100 10857 -456 99 -130 65 -362 391 -2866 99 -1128 99 -334 365 -198 199 -300 65 -166 133 -300 331 -66 297 -66 233 -166 295 -524 199 -98 65 -132 65 -296 329 -132 99 -134 369 -100
RAW_Data: 799 -66 6183 -230 65 -392 63 -132 163 -592 131 -296 197 -262 195 -3012 199 -66 131 -298 65 -200 97 -298 165 -166 303 -132 1027 -198 167 -100 263 -66 165 -100 363 -66 263 -102 169 -66 957 -66 163 -164 7961 -164 393 -1194 67 -234 67 -594 99 -460 97 -1216 97 -166 235 -730 99 -200 99 -630 65 -168 301 -100 497 -98 297 -66 929 -68 3295 -66 8639 -232 63 -4578 65 -266 201 -2586 329 -298 163 -198 229 -66 263 -200 265 -166 199 -198 961 -132 993 -66 663 -166 593 -132 231 -200 9507 -1784 131 -1236 231 -728 131 -100 163 -100 261 -594 97 -132 523 -888 65 -328 263 -132 163 -100 1227 -132 357 -100 393 -132 67 -232 497 -232 233 -100 199 -200 299 -100 201 -98 8169 -1504 97 -530 65 -556 65 -464 97 -4638 99 -2656 67 -364 65 -98 131 -164 161 -98 261 -132 163 -722 97 -132 263 -988 295 -166 163 -164 559 -98 691 -298 12159 -752 97 -392 261 -294 65 -130 65 -66 131 -558 99 -132 299 -66 231 -134 131 -332 167 -66 299 -66 231 -100 199 -568 365 -1160 231 -198 131 -66 199 -100 65 -134 231 -168 331 -102 759 -100 163 -264 525 -66 11151 -1230 65 -2476 65 -298 393 -66 1419 -132 231 -68 767 -362 99 -300 299 -266 97 -100 501 -134 1487 -66 131 -132 5737 -1956 65 -630 65 -1986 165 -98 229 -66 197 -98 97 -132 391 -162 229 -198 65 -1914 397 -134 133 -132 133 -66 263 -232 361 -132 1029 -98 1415 -98 9837 -464 99 -364 165 -166 133 -134 99 -98 133 -362 263 -132 459 -394 97 -98 131 -494 131 -300 233 -732 463 -134 197 -168 2055 -100 8457 -6368 65 -134 233 -894 231 -398 65 -66 299 -98 267 -170 65 -68 233 -66 271 -100 599 -532 463 -330 297 -400 199 -168 99 -164 467 -66 233 -170 10505 -66 67 -630 163 -788 99 -66 197 -260 99 -98 295 -66 195 -98 195 -856 197 -134 297 -132 199 -654 331 -132 327 -394 131 -164 165 -266 629 -166 133 -66 233 -134 131 -398 99 -1030 759 -166 299 -102 365 -164 363 -98 165 -166 231 -300 363 -100 263 -200 99 -66 263 -166 65 -68 201 -198 233 -134 793 -234 263 -100 261 -64 229 -66 261 -162 2055 -430 231 -5476 197 -1894 297 -166 99 -134 333 -662 131 -530 199 -298 197 -200 233 -66 201 -166 297 -100 1317 -100 685 -132 229 -3476 133 -396 231 -166 927 -134 299 -100 467 -100 231 -100 101 -100 231 -234 399 -166 167 -166 827 -132 435 -68 435 -66 867 -100 7831 -1786 67 -3442 65 -860 99 -1028 65 -964 497 -100 229 -132 295 -98 329 -98 427 -132
RAW_Data: 197 -162 297 -132 519 -66 7671 -458 129 -524 65 -528 163 -926 97 -362 65 -264 65 -262 197 -690 99 -298 331 -332 231 -432 393 -332 627 -168 199 -66 333 -100 101 -134 363 -66 131 -136 65 -268 1453 -98 11127 -134 99 -296 129 -2336 261 -132 589 -66 129 -132 495 -134 563 -230 825 -100 1427 -298 661 -98 819 -130 163 -66 393 -98 99 -164 263 -394 361 -100 459 -64 13711 -912 97 -628 97 -66 131 -196 131 -492 65 -628 99 -424 427 -66 199 -134 367 -198 131 -68 295 -100 331 -200 167 -396 163 -300 167 -330 99 -98 427 -100 397 -100 1065 -132 7655 -1096 65 -292 97 -132 131 -4634 131 -302 133 -68 267 -364 295 -164 99 -130 95 -132 1317 -98 195 -166 399 -100 267 -100 431 -230 199 -200 99 -200 265 -132
```

Button 2:
```
Filetype: Flipper SubGhz RAW File
Version: 1
Frequency: 433657070
Preset: FuriHalSubGhzPresetOok650Async
Protocol: RAW
RAW_Data: 13169 -164 163 -1086 65 -200 163 -100 2311 -434 165 -100 431 -166 65 -1490 197 -198 269 -102 233 -100 231 -102 303 -134 167 -164 425 -162 761 -68 531 -132 399 -234 65 -134 427 -362 231 -228 65 -96 1741 -98 7309 -8136 363 -234 197 -296 263 -1064 265 -132 201 -168 97 -132 165 -232 633 -66 593 -166 857 -64 5195 -604 133 -2768 65 -5138 65 -528 429 -428 97 -132 229 -132 163 -100 261 -294 427 -64 263 -228 97 -98 263 -132 297 -64 357 -66 231 -96 291 -132 161 -98 265 -162 2105 -102 6717 -1030 131 -930 131 -722 327 -130 393 -130 261 -132 165 -132 97 -428 97 -100 129 -758 197 -426 65 -132 97 -162 329 -194 63 -462 929 -68 265 -132 6503 -236 67 -500 65 -1432 65 -3106 65 -1798 131 -100 233 -434 97 -166 231 -200 99 -202 399 -166 1887 -164 399 -66 597 -198 67 -66 299 -166 791 -66 6599 -194 97 -96 263 -498 131 -756 229 -6182 65 -1118 99 -426 325 -228 359 -162 985 -100 65 -64 99 -64 99 -232 333 -164 65 -98 129 -296 11589 -198 65 -4880 263 -262 129 -130 97 -196 395 -164 231 -134 197 -300 99 -166 267 -132 297 -496 393 -132 297 -164 195 -228 131 -166 99 -996 199 -130 299 -166 267 -134 263 -132 1269 -100 465 -100 8591 -136 135 -632 131 -2906 133 -332 131 -266 99 -332 65 -102 333 -134 133 -430 297 -168 299 -196 65 -98 329 -64 197 -132 195 -396 589 -98 363 -166 331 -164 165 -98 265 -98 293 -98 11619 -1544 65 -3162 97 -98 201 -134 99 -698 199 -2718 65 -134 7831 -232 65 -1196 131 -936 67 -166 67 -6204 195 -660 65 -396 297 -68 133 -166 97 -200 399 -100 167 -366 231 -64 265 -496 197 -230 131 -164 131 -164 97 -394 65 -530 427 -628 197 -130 1245 -100 4145 -198 461 -1258 591 -200 263 -98 197 -162 65 -950 165 -762 65 -168 67 -66 233 -198 199 -730 7487 -6950 97 -362 457 -66 297 -164 199 -166 331 -66 657 -234 165 -134 165 -168 429 -2610 267 -132 233 -100 765 -100 827 -66 883 -8336 199 -2268 295 -690 65 -430 231 -898 131 -1524 133 -366 233 -132 133 -232 133 -164 231 -134 99 -66 563 -68 959 -134 2823 -10148 65 -264 65 -64 331 -168 267 -794 265 -132 99 -294 497 -66 597 -102 165 -234 433 -130 635 -298 231 -902 131 -166 131 -1798 165 -1026 231 -132 265 -98 333 -168 99 -596 265 -166 101 -134 201 -66 201 -464 65 -100 163 -132 97 -98 163 -264 197 -98 229 -232 97 -132 719 -228 263 -100 271 -134 6071 -8314 99 -664 67 -1494 265 -168 199 -166 99 -200 163 -132
RAW_Data: 133 -200 99 -168 133 -100 297 -232 295 -98 163 -130 65 -162 163 -294 263 -428 327 -1768 65 -4372 131 -198 99 -2288 163 -430 231 -134 131 -236 67 -132 1433 -66 131 -234 65 -100 399 -168 527 -168 233 -66 331 -198 8139 -1084 65 -598 129 -560 65 -4830 229 -430 97 -100 133 -68 265 -264 99 -66 65 -166 1793 -132 165 -100 731 -68 529 -132 529 -66 7117 -296 99 -7912 197 -200 165 -202 265 -298 231 -132 99 -68 855 -264 329 -132 161 -196 195 -856 65 -98 199 -232 165 -132 6941 -8028 765 -926 65 -632 65 -100 465 -232 197 -238 531 -132 131 -134 99 -300 399 -198 131 -166 133 -132 465 -66 1659 -2402 65 -7428 165 -392 229 -100 723 -198 63 -328 99 -826 65 -1268 65 -334 331 -628 297 -100 491 -164 1557 -100 3297 -2352 131 -2346 65 -130 99 -920 263 -492 99 -132 99 -864 231 -132 559 -100 6687 -9048 99 -464 197 -960 131 -98 131 -360 99 -364 65 -2700 233 -100 99 -232 131 -598 99 -66 297 -132 263 -68 1095 -100 1281 -368 163 -3820 133 -1496 165 -1792 299 -166 131 -134 297 -764 265 -330 8635 -234 133 -2920 99 -1820 99 -1818 229 -134 265 -630 133 -364 233 -198 927 -100 819 -66 5749 -6118 131 -66 197 -726 295 -132 99 -134 163 -1060 163 -1052 231 -164 197 -132 391 -130 259 -496 499 -98 11683 -400 199 -758 131 -1158 165 -166 197 -196 97 -198 393 -68 97 -132 523 -66 557 -132 361 -362 653 -166 459 -230 229 -198 837 -322 631 -338 633 -348 605 -358 637 -334 603 -348 639 -326 641 -302 665 -318 635 -328 641 -302 665 -316 637 -326 641 -332 633 -318 635 -5144 635 -328 643 -312 655 -310 649 -294 667 -316 659 -298 649 -346 621 -314 649 -294 667 -318 659 -298 649 -346 621 -314 649 -324 639 -318 623 -5154 635 -324 661 -314 657 -310 651 -310 625 -352 625 -294 665 -318 659 -298 649 -310 655 -314 661 -294 667 -318 629 -328 649 -312 655 -312 649 -4818 293 -630 351 -624 335 -320 609 -644 349 -628 293 -636 353 -318 625 -634 349 -626 299 -652 311 -656 311 -322 631 -654 321 -630 325 -646 311 -624 339 -622 345 -624 313 -324 637 -644 319 -638 319 -328 649 -616 337 -322 627 -294 667 -624 323 -642 351 -284 659 -610 347 -624 337 -624 345 -620 313 -322 641 -320 659 -622 319 -626 349 -320 625 -312 649 -614 343 -324 639 -290 649 -634 321 -24198 315 -628 329 -650 311 -304 669 -622 323 -640 317 -626 329 -314 663 -612 343 -624 345 -624 347 -626 293 -310 669 -624 327 -648 311 -624 341 -626 343 -626 311 -650 323 -312
RAW_Data: 641 -622 329 -650 311 -306 665 -624 323 -314 645 -310 657 -648 311 -624 339 -322 639 -610 353 -626 321 -640 309 -634 351 -286 671 -290 669 -622 321 -626 349 -322 623 -312 661 -616 337 -322 633 -292 669 -622 321 -24192 313 -630 339 -616 317 -342 639 -624 327 -646 315 -632 351 -278 663 -614 347 -640 317 -630 327 -646 309 -306 667 -622 323 -642 317 -626 331 -650 311 -624 343 -624 345 -302 641 -636 321 -628 349 -320 625 -646 313 -304 667 -320 631 -616 355 -630 321 -306 649 -612 343 -624 345 -624 349 -628 293 -308 669 -314 655 -610 349 -622 335 -322 627 -292 667 -624 323 -314 645 -312 655 -646 313 -24198 313 -614 319 -668 319 -318 625 -634 351 -624 299 -652 309 -336 641 -640 289 -652 349 -624 335 -620 343 -302 637 -640 319 -624 349 -624 335 -620 345 -622 311 -658 309 -302 649 -654 287 -652 313 -312 649 -654 321 -326 621 -344 621 -648 311 -656 307 -322 639 -646 317 -626 321 -646 315 -628 329 -348 609 -320 645 -654 289 -650 313 -354 623 -312 645 -646 309 -322 635 -322 623 -654 321 -24194 315 -632 325 -644 311 -304 667 -622 323 -642 311 -634 351 -320 633 -616 353 -630 289 -670 311 -624 341 -320 625 -644 311 -660 293 -670 319 -628 323 -644 309 -624 341 -320 625 -644 313 -646 325 -312 645 -622 327 -350 607 -320 645 -654 289 -650 315 -310 649 -638 329 -648 311 -654 311 -658 307 -304 647 -314 659 -630 317 -626 329 -348 623 -294 669 -622 323 -314 643 -346 621 -646 311 -24190 303 -652 311 -658 307 -322 633 -656 319 -628 325 -642 311 -304 667 -622 323 -640 311 -634 351 -624 323 -312 641 -654 297 -650 309 -656 311 -658 307 -628 347 -626 293 -342 639 -622 329 -646 309 -306 669 -620 323 -316 643 -344 621 -648 311 -656 307 -322 625 -644 349 -624 293 -668 311 -634 349 -320 601 -322 665 -620 321 -628 349 -318 625 -312 645 -640 311 -322 633 -322 629 -654 321 -24194 313 -632 327 -648 311 -304 665 -622 321 -644 315 -660 297 -352 609 -646 311 -660 293 -668 319 -628 325 -310 645 -646 311 -658 309 -624 349 -626 293 -670 309 -636 347 -322 599 -648 317 -660 289 -352 619 -654 297 -352 609 -324 645 -638 319 -628 349 -318 625 -648 311 -656 309 -648 309 -626 349 -322 631 -290 667 -620 323 -664 313 -320 625 -312 647 -644 311 -320 661 -290 645 -638 325 -24212 313 -626 299 -646 317 -312 667 -622 327 -644 303 -662 319 -310 633 -646 315 -636 347 -634 297 -646 317 -310 661 -640 299 -648 315 -636 349 -634 297 -650
RAW_Data: 309 -658 311 -320 639 -644 319 -624 323 -318 645 -652 299 -352 607 -324 645 -654 289 -650 313 -312 651 -654 321 -622 347 -624 335 -620 345 -300 641 -316 627 -658 311 -634 347 -322 631 -290 669 -626 327 -308 645 -318 639 -654 287 -112852 131 -100 65 -132 99 -736 65 -298 65 -66 165 -1756 331 -1200 99 -198 167 -168 263 -134 433 -132 301 -100 2191 -202 9691 -200 165 -66 131 -134 99 -168 265 -98 229 -296 65 -130 99 -132 131 -130 361 -130 1345 -262 523 -162 229 -66 231 -926 195 -358 99 -526 329 -232 1929 -300 891 -98 787 -66 3449 -8696 265 -298 163 -1252 97 -100 65 -528 265 -200 99 -3312 133 -132 165 -166 65 -864 627 -98 163 -132 9997 -4380 133 -66 229 -454 97 -494 99 -1282 195 -1454 197 -668 229 -132 65 -66 331 -66 263 -168 299 -66 331 -166 6551 -8294 131 -500 97 -1630 97 -66 99 -66 263 -200 133 -364 65 -828 67 -166 67 -532 99 -264 329 -100 199 -198 461 -98 97 -98 195 -164 165 -132 423 -198 2791 -66 6841 -498 131 -658 131 -722 163 -794 65 -624 165 -658 131 -262 97 -230 97 -66 163 -132 129 -228 99 -460 459 -526 263 -1032 229 -66 163 -66 885 -98 6869 -9164 1385 -232 501 -896 131 -200 133 -268 401 -66 201 -132 199 -100 363 -66 869 -9434 65 -1162 99 -494 197 -294 97 -662 131 -362 265 -1254 65 -66 531 -136 201 -164 199 -68 197 -100 163 -132 495 -100 165 -132 231 -66 5221 -8518 263 -1020 133 -630 231 -502 133 -998 199 -98 199 -728 97 -630 231 -198 591 -66 5507 -10944 263 -1594 367 -166 133 -134 163 -200 99 -168 5539 -9732 99 -2700 167 -1090 65 -1248 131 -98 195 -754 65 -100 361 -132 259 -66 1471 -132 1969 -1586 131 -268 129 -494 65 -132 65 -594 97 -66 231 -66 261 -132 65 -526 163 -194 95 -100 459 -690 197 -196 97 -132 297 -568 99 -134 131 -630 329 -100 299 -332 65 -100 593 -64 361 -66 301 -100 1439 -66 9677 -792 197 -760 325 -166 65 -296 99 -362 129 -132 227 -64 197 -198 163 -886 65 -1088 65 -392 65 -98 263 -66 229 -296 1083 -98 359 -164 2809 -66 2815 -64 457 -5460 97 -856 133 -166 65 -302 533 -66 163 -164 133 -98 299 -168 371 -166 99 -66 265 -66 4171 -64 881 -10358 265 -656 165 -328 263 -98 99 -334 365 -720 131 -300 265 -332 469 -132 265 -100 401 -330 165 -194 625 -162 529 -66 295 -130 63 -230 3655 -8286 133 -794 131 -1288 131 -98 131 -564 267 -166 67 -134 99 -100 65 -100 165 -198 65 -298 65 -198 199 -498 99 -66 131 -132
RAW_Data: 99 -66 229 -194 557 -266 131 -166 133 -98 661 -13170 65 -560 65 -268 131 -266 135 -134 99 -298 67 -198 533 -268 495 -134 131 -298 97 -98 65 -98 199 -390 65 -426 97 -164 225 -130 953 -332 295 -66 1919 -8412 65 -2090 263 -100 297 -730 99 -134 99 -566 101 -198 165 -458 99 -198 889 -164 163 -130 263 -196 8435 -4188 131 -3304 99 -130 97 -588 67 -198 99 -102 133 -334 197 -296 65 -164 95 -328 197 -198 99 -264 97 -262 97 -132 263 -262 163 -696 229 -200 199 -98 165 -134 601 -166 5443 -11390 165 -864 97 -630 131 -132 65 -494 759 -232 131 -134 8353 -4156 101 -1198 65 -1288 361 -264 97 -1490 231 -364 297 -134 267 -566 65 -132 197 -330 265 -198 459 -100 263 -164 885 -66 2693 -8540 99 -828 99 -1296 229 -166 65 -332 65 -234 67 -1824 195 -2108 99 -394 363 -98 259 -98 459 -164 131 -914 131 -2660 165 -2502 65 -866 131 -360 233 -100 297 -134 297 -68 331 -232 65 -334 1229 -100 5977 -198 97 -3696 397 -64 99 -1494 131 -598 97 -952 131 -1290 65 -866 97 -932 231 -1430 265 -920 525 -132 795 -132 501 -132 9277 -2128 63 -1082 97 -496 97 -100 167 -662 63 -2148 199 -102 267 -200 231 -164 165 -398 67 -98 99 -100 3909 -4362 97 -1456 233 -66 65 -1360 263 -100 261 -298 165 -1030 65 -492 163 -98 65 -2478 133 -268 65 -134 565 -68 165 -366 329 -228 231 -232 7081 -4160 165 -132 65 -2406 165 -1324 265 -98 199 -132 235 -1198 165 -166 233 -132 167 -458 229 -1484 229 -800 425 -164 433 -68 1593 -132 7055 -4980 65 -594 97 -66 163 -1518 263 -130 65 -956 263 -296 231 -98 1153 -166 65 -366 5449 -2298 97 -6498 65 -1452 165 -230 327 -132 259 -66 65 -98 99 -558 165 -162 65 -656 65 -430 299 -98 199 -100 331 -66 333 -100 357 -194
```

**Comment by Skorpionm** [comment]
I repeat once again you do not have a stock version of the firmware, put the stock one and record 5 presses of each button and holding for 3 seconds

**Comment by Eugst** [comment]
Facing the same trouble. Firmware from the box. Tried 5 times per 3 second - didn't help.

**Comment by skotopes** [comment]
Issue was fixed in #2443 please check and let us know if it still persist
